migrate_pim/upgrade/upgrade_from_40_to_50.rst

```diff
@@ -29,6 +29,52 @@ You have to make sure your system components are updated to the version required
 
     So there's no need to export and reimport data for this system.
 
+
+Updated System dependencies
+---------------------------
+
+Community Edition
+^^^^^^^^^^^^^^^^^
+TODO?
```

Comment:
Who might know about this?

Reply:
If no one logged it all along the year, I would say a git diff off the dockerfile between 4.x and master could be a good start.

Reply:
I finally understood what  you meant...    
Main diff in Dockerfile.base is:
apt-get --yes install apt-transport-https ca-certificates curl wget &&\
    wget -O /etc/apt/trusted.gpg.d/php.gpg https://packages.sury.org/php/apt.gpg &&\